invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -232,22 +232,48 @@ def view(**kwargs):
             if is_preview:
                 try:
                     files = draft_files_service().list_files(**read_kwargs)
-                    media_files = draft_media_files_service().list_files(**read_kwargs)
```

Comment:
`files` and `media_files` have different permission logic, so for example failing with `PermissionDenied` for files doesn't have to result with `kwargs["media_files"] = None` as well. Moved to separate decorator